SL1_to_Photon.py

```diff
@@ -62,12 +64,36 @@ def extract_images(self, dirpath):
         print('ERROR: file {} already exists!. move or use -f flag to force overwrite. Cancelling...'.format(os.path.basename(photon_path)))
         sys.exit(-1)
 
+    print(args.liftspeed)
+
     sl1 = SL1Reader(args.sl1_file)
     photon = pyphotonfile.Photon()
     photon.exposure_time = float(sl1.config['expTime'])
     photon.exposure_time_bottom = float(sl1.config['expTimeFirst'])
     photon.layer_height = float(sl1.config['layerHeight'])
     photon.bottom_layers = int(sl1.config['numFade'])
+    photon.bottom_layer_count = int(sl1.config['numFade'])
+    photon.lifting_speed = int(args.liftspeed)
+    photon.bottom_lift_speed = int(args.liftspeed)
+    photon.version = 2;
+    photon.print_properties_length = 60
+    photon.retract_speed = int(args.retractspeed)
+    photon.print_time = int(float(sl1.config['printTime']))
```

Comment:
All of those settings do not fit this PR, and require explanation.

Reply:
Ooof, sorry, didn't update pyphotonfile prior to review — everything is clearer now)

Reply:
Merged that into my fork